### DISCUSSION TITLE
entitie definations

### DISCUSSION THREAD

**Discussion by myxtrastufff** [WillCodeForCats/solaredge-modbus-multi#106] [Ideas]
Hello, Thank you very much for your work on this integration! I am new to HA and I have been trying to figure all this out. Are the below entities the same that I would get from other integrations that use SolarEdge modbus or are they provided by you? Could you define what all the entities do? I am trying to understand how to use these in HA. I see another integration that you commented on where they use a template to create sensors get more use out of the integration. Is a template needed with yours to provide more info? For example, total solar production, total self usage, and import/export from/to the grid. If those numbers are something you could build into this, that would be awesome. Or am I not seeing them? If not, I understand enough how to use a sensor template to make it work. I am basically looking to use this to charge the car on only excess solar and use the "tesla style" card in HA. I know everyone's inverter setup will be different. I have 3 single phase inverters and for some reason only two show up in your integration. I am still trying to figure out why only two show up. But even if I could get two of the three that would be enough to charge the car. Thanks again!

sensor.solaredge_i1_ac_current   10A
sensor.solaredge_i1_ac_current_a 10A
sensor.solaredge_i1_ac_current_b
sensor.solaredge_i1_ac_current_c
sensor.solaredge_i1_ac_energy_kwh 26,158kWh total produced?
sensor.solaredge_i1_ac_frequency  59.9Hz
sensor.solaredge_i1_ac_pf
sensor.solaredge_i1_ac_power 2,370W house usage?
sensor.solaredge_i1_ac_va
sensor.solaredge_i1_ac_var
sensor.solaredge_i1_ac_voltage_ab 237V
sensor.solaredge_i1_ac_voltage_an
sensor.solaredge_i1_ac_voltage_bc
sensor.solaredge_i1_ac_voltage_bn
sensor.solaredge_i1_ac_voltage_ca
sensor.solaredge_i1_ac_voltage_cn
sensor.solaredge_i1_dc_current 6.355A panel production?
sensor.solaredge_i1_dc_power 2,400W panel production?
sensor.solaredge_i1_dc_voltage 378V
sensor.solaredge_i1_device Model#
sensor.solaredge_i1_status 4?
sensor.solaredge_i1_status_vendor 0?
sensor.solaredge_i1_temp_sink 126F Heat sink Tempature
sensor.solaredge_i1_version Firmware version?

**Comment by WillCodeForCats** [comment]
Without additional meters the only data you can get from an inverter is production in different forms (DC in from panels, AC out from inverter, amps, volts, energy, etc.) If you do not have any meters in your installation and want grid export+import or self-consumption, then you will need to install meters for those since they cannot be determined from inverter data alone.

As far as other inverters not showing up, make sure you added the integration configured for 3 inverters, and your inverters are installed and configured on the RS485 chain with one leader (the one connected to Ethernet) and two followers with incrementing device IDs.

For more information on configuration see:
https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
